kafka/topics.go

```diff
@@ -1,3 +1,3 @@
-//Topics, should be include to kafka: positions, matched-positions.
-
+// Package kafka provides kafka connection.
+// Topics, should be include: positions, matched-positions.
```

Comment:
This row is about external stuff. Remove it.

Reply:
Yep